packages/story-editor/src/utils/getInUseFonts.ts

```diff
@@ -14,20 +14,24 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import type { Page } from '@googleforcreators/elements';
+import type { TextSet } from '@googleforcreators/text-sets';
+import { elementIs } from '@googleforcreators/elements';
+
 /**
  * Returns a list of font names used across all pages in a story.
- *
- * @param {Array<import('../types').Page>} pages List of pages.
- * @return {Array<string>} Font names.
  */
-export function getInUseFontsForPages(pages) {
+export function getInUseFontsForPages(pages: Page[]) {
   return (
     Array.from(
       new Set(
         pages
           .map(({ elements = [] }) =>
             elements.map((element) => {
-              if (element.type === 'text' && Boolean(element.font?.family)) {
+              if (elementIs.text(element) && Boolean(element.font?.family)) {
```

Comment:
Same here, `font.family` is required for text element.

Reply:
Fixed.